### ISSUE TITLE
[Feature] Improved voiceover on journey details

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#622]
## Acceptance criteria 

- [ ] Screen reading of elements in details matches [Figma script] (or close enough to be acceptable) (https://www.figma.com/file/KTz9Bzj29zA22uzgTJXKS1/0.9-Accessibility-descriptions?node-id=11%3A43)
- [ ] Reading, pausing, is understandable (voiceover on iOS not tested by developer)

 ## Origin
_Links to received feedback, user research or other findings._
This issue depends on #748, since the new voiceover script is based on the new design.

[Findings from usability test with Blindeforbundet 29.10.2020](https://github.com/AtB-AS/innsikt/blob/master/brukertester/2020-10-29_Blindeforbundet.md)

> Ved bruk av voiceover på reisedetaljer kom det frem viktigheten av å få opp informasjon om hvor mange mellomstopp det er, hvilke stopp man skal bytte på og hvor man skal gå av.

## Motivation
_A short description of what user needs or business goals this feature will solve._

## Hypotheses and assumptions
_A list of hypotheses and assumptions we have made about the user or the proposed solution._

## Proposed solution
_A coarse description of a proposed solution, that may include wireframes and graphic design._

Update screen reading according to script in [Figma](https://www.figma.com/file/KTz9Bzj29zA22uzgTJXKS1/0.9-Accessibility-descriptions?node-id=11%3A43)

**Comment by rikkekantega** [comment]
This issue depends on #748, since the new voiceover script is based on the new design.

**Comment by mikaelbr** [comment]
QA:

- Deactivated buttons (e.g. paging on journey details) aren't registered as deactivated by voice over. No indication that it is an action that cannot be triggered.
- Read `1 steg` instead of `Første steg` on iOS ScreenReader. Not critical and is ideally something that iOS should handle when text is `1.`, but can be confusing for user.
- "Gangavstand" should it also read "Total gangavstand"?

**Comment by rikkekantega** [comment]
> QA:
> 
> * Deactivated buttons (e.g. paging on journey details) aren't registered as deactivated by voice over. No indication that it is an action that cannot be triggered.
> * Read `1 steg` instead of `Første steg` on iOS ScreenReader. Not critical and is ideally something that iOS should handle when text is `1.`, but can be confusing for user.
> * "Gangavstand" should it also read "Total gangavstand"?
> 
> cc @rikkekantega My observations above. I don't know how you want to handle findings. Either as separate tasks or just ignore/prioritize.

1. I'll fix the button component for deactivated state.
2. Bummer that iOS Voiceover doesn't read out "1." as "Første". My suggestion is to change "1. Steg" to: "Steg 1" 
3. Does it read "Gangavstand" _and_ "Total gangavstand"?

As to handling, when in QA mode I'd like to resolve "minor" findings like these rather quickly per original issue.
When QA results in _new_ tasks, things that weren't part of the original task/acceptance criteria, I'd rather create a separate task for it.

**Comment by mikaelbr** [comment]
3. No, I was thinking that since for "Reisetid:" it reads "Total reisetid" it should also do that for "Gangavstand".

**Comment by rikkekantega** [comment]
> 1. No, I was thinking that since for "Reisetid:" it reads "Total reisetid" it should also do that for "Gangavstand".

Ah I see. Sure, that makes sense. :)
